cli.js

```diff
@@ -142,6 +142,8 @@ process.on('uncaughtException', function(err) {
 
     flags = flags || [];
     optFlags = optFlags || [];
+    // we always take a log option
+    optFlags.push('log');
```

Comment:
I guess global arguments will always be a thing though, so the architecturally correct solution would probably be to wrap `readOptions` with some default global handlings within a new function closure.